Robots 101 section (#527)
srobo/docs: robots_101/team_supervisor.md

---
layout: page
title: Robots 101 - So you're running a team
---

# Robots 101 - So you're running a team

Here’s a little bit of info on what to expect from Student Robotics, and your responsibilities as a team supervisor.

As a team supervisor, your role is to guide the competitors through the journey of building a robot. You’ll be there to point them in the right direction when they get stuck and resolve any issues they run into. We encourage team supervisors to limit their involvement with the robot design/building process so that the finished contraptions are 100% student-built which competitors find very rewarding.

> You’ll be our point of contact with the team. If you have any questions during the year, just email [teams@studentrobotics.org](mailto:teams@studentrobotics.org). We’ll also be sending you emails every month or so with important information such as:

Reviewer: ```suggestion
You’ll be our point of contact with the team. If you have any questions during the year, just email <{{ site.emails.teams }}>. We’ll also be sending you emails every month or so with important information such as:
```

- Details of Tech Days where we provide a space for teams to come together and get direct help from our dedicated volunteers
- Software updates for our kit

> - Information about the competition in April

Reviewer: ```suggestion
- Information about the competition event (usually in late March or April)
```

This is especially important to book for the competition, as you will likely need to arrange to stay overnight near the venue.

Our kit includes a battery and a few boards to get your robot started. However, your team will need additional components and materials from which to build their robot. For the chassis, your team has several options. Teams often use cardboard, MDF, Aluminium, and/or Acrylic. In terms of electronic components, teams often make use of:
- At least two 12v motors (our kit supports four)
- A few servo motors
- A few microswitches for detecting if you bump into something
- Ultrasound sensor
- Wheels & bearings

Be sure to plan with your team what your robot will need before you make any purchases as depending on your robot design you may need more or less of these components.

Popular suppliers of these components are:
- [Robot Shop](https://www.robotshop.com/)
- [RS](https://uk.rs-online.com/web/)
- [Farnell](https://uk.farnell.com/)
- [Amazon](https://www.amazon.co.uk/)
- [eBay](https://www.ebay.co.uk/)

> To allow you and your team to ask us questions directly, as well as share what they’re working on with other teams, we provide a [Discord]({{ site.baseurl }}/team_admin/discord) server. You will receive a unique link to share with your team so that you can all join with your own Discord accounts. Each team gets a private text channel for direct support from us which can be used to discuss your robot without worrying about giving other teams your strategy. We also have a team-supervisor only channel for any questions you may have. There are also some text channels where teams can communicate with each other (and us!) for more general topics. If you’d like a voice channel for your team, email us or ask in `#team-supervisor`!

Reviewer: ```suggestion
To allow you and your team to ask us questions directly, as well as share what they’re working on with other teams, we provide a [Discord]({{ site.baseurl }}/tutorials/discord) server. You will receive a unique link to share with your team so that you can all join with your own Discord accounts. Each team gets a private text channel for direct support from us which can be used to discuss your robot without worrying about giving other teams your strategy. We also have a team-supervisor only channel for any questions you may have. There are also some text channels where teams can communicate with each other (and us!) for more general topics. If you’d like a voice channel for your team, email us or ask in `#team-supervisor`!
```
